2-With-broker/UserDetailsClient/UserDetailsClient/MainPage.xaml.cs

```diff
@@ -10,53 +11,33 @@
 namespace UserDetailsClient
 {
     public partial class MainPage : ContentPage
-    {        
+    {
         public MainPage()
         {
-            InitializeComponent();
+            InitializeComponent();
+            CreatePublicClient();
         }
 
-        private void OnSignInSignOut(object sender, EventArgs e)
-        {            
-            CreatePublicClient(false);
-            AcquireTokenAsync().ConfigureAwait(false);
-        }
-
-        public static void CreatePublicClient(bool useBroker)
+        public static void CreatePublicClient()
         {
-            var builder = PublicClientApplicationBuilder
-                .Create(App.ClientID);
-                
-            if (useBroker)
-            {                
-                switch (Device.RuntimePlatform)
-                {
-                    case Device.Android:
-                        builder = builder.WithRedirectUri(App.BrokerRedirectUriOnAndroid);
-                        break;
-                    case Device.iOS:
-                        builder = builder.WithIosKeychainSecurityGroup("com.microsoft.adalcache");
-                        builder = builder.WithRedirectUri(App.BrokerRedirectUriOnIos);
-                        break;
-                    case Device.UWP:
-                        builder = builder.WithExperimentalFeatures();
-
-                        // See also UserDetailsClient.UWP project in MainPage.xml.cs
-                        // To get the redirect URI that you need to register in your app
-                        // registration of a shape similar to:
-                        // ms-appx-web://microsoft.aad.brokerplugin/S-1-15-2-3163378744-4254380357-4090943427-3442740072-2185909759-2930900273-1603380124
-                        builder.WithDefaultRedirectUri();
-                        break;
-                }
-
-                builder.WithBroker();
-            }
-            else
+            string redirectUri = null;
+            switch (Device.RuntimePlatform)
             {
-                builder = builder.WithRedirectUri($"msal{App.ClientID}://auth");
+                case Device.Android:
+                    redirectUri = App.BrokerRedirectUriOnAndroid;
+                    break;
+                case Device.iOS:
+                    redirectUri = App.BrokerRedirectUriOnIos;
+                    break;
             }
 
-            App.PCA = builder.Build();
+            PCAHelper.Init<PCAHelper>(App.ClientID, App.Scopes, redirectUri);
+            if (Device.RuntimePlatform == Device.UWP)
+            {
+                PCAHelper.Instance.PCABuilder.WithExperimentalFeatures();

```

Comment:
WAM is no longer under experimental features, I think.

Reply:
Kept it consistent with the original sample code. I will remove, if not required. @pmaytak  https://github.com/Azure-Samples/active-directory-xamarin-native-v2/blob/e685e98eb577a96073e9c5c4137fec76340fafb2/2-With-broker/UserDetailsClient/UserDetailsClient/MainPage.xaml.cs#L42

Reply:
Correct, it is no longer experimental.